ikenna-portfolio/portfolio/src/main/webapp/script.js

```diff
@@ -12,160 +12,76 @@
-  Although I technically succeeded in doing such I found out that based on the way opengl (a graphics library) renders its data, \
-  I am not allowed to render across multiple threads. The fix for this would be a synchronization at the end of each step of \
-  the the game state, but that seemed like an unrewarding solution. So, to this day, I am thinking of better solutions/trying to \
-  see how I can abstract the messaging system away from the game engine to make it easier to implement everything else. The \
-  abstraction would also allow me to implement other projects I might want multithreaded such as a genetic algorithm builder \
-  or neural net generator."
-];
-
-const links = [
-  "Here's a link to the <a href=\"https://arxiv.org/pdf/1703.10593.pdf\">paper</a>.",
-
-  "Here's a link to the <a href=\"https://arxiv.org/pdf/1703.07511.pdf\">paper</a>.<br> \
-  Here's a link to the pytorch <a href=\"https://pytorch.org/tutorials/advanced/neural_style_tutorial.html\">tutorial</a>.", 
-      
-  "Here's a link to the <a href=\"https://www.gamasutra.com/blogs/MichaelKissner/20151027/257369/Writing_a_Game_Engine_from_Scratch__Part_1_Messaging.php\">site</a> explaining the message system further."
-];
+const project_name = "<small>Project Showing: -</small>";
 
 const goal = "Goal: -";
 const current = "Current Ability: -";
 
-/**
-* Initializes the page with containers and server requests
-*/
-async function initializePage() {
-  const response = await fetch('/data');
-  const msg = await response.json();
-  let entire_msg = "";
-  for(m of msg) {
-    entire_msg += "<p>" + m + "</p>";
-  }
-  document.getElementById('hello-world').innerHTML = entire_msg;
-  loadInteractiveContainers();
-}
-
-/**
-* Changes Projects and Calisthenics Containers to their default states
-*/
-function loadInteractiveContainers() {
-  changeCycleGAN();
-  changeVSit()
-}
-
-/**
-* Adds a random quote to the page.
-*/
-async function addRandomQuote() {
-  const response = await fetch('/random');
-  const quote = await response.text();
-  document.getElementById('quote-container').innerText = quote;
-}
-
-/**
-* Changes Projects Container to CycleGAN state
-*/
-function changeCycleGAN() {
-  document.getElementById("pro-image1").src = "/images/real.png";
-  document.getElementById("pro-image2").src = "/images/real_to_rot.png";
-  document.getElementById("pro-image3").src = "/images/rot_to_real.png";
-  document.getElementById("pro-desc").innerHTML = project_desc[0];
-  document.getElementById("pro-links").innerHTML = links[0];
-}
+const comment_item = "<li class=\"media\"><div class=\"media-body\"><button name=\"delete-comment\" class=\"pull-right btn btn-secondary\" value=\"comment_id\">x</button><small class=\"pull-right\">timestamp</small><strong class=\"pull-left\">comment_name</strong><br><br><p class=\"desc\" align=\"left\">comment_text</p></div></li>";
```

Comment:
same as above as it's usually good to separate the static and dynamic contents.

Reply:
Since I'm adding in the comments dynamically, I do not know how to put it in the original html (since I don't know the size beforehand). Do you have an idea of how to do it statically while adding comments dynamically?

Reply:
I got it. This is a template for one comment and there could be multiple ones.